.github/workflows/ci.yml

```diff
@@ -94,3 +94,31 @@ jobs:
         comment-id: ${{ steps.find_comment.outputs.comment-id }}
         issue-number: ${{ github.event.pull_request.number }}
         token: ${{ secrets.GITHUB_TOKEN }}
+  sdk-compile-test:
+    name: "Compile target:${{ matrix.target }} compile:${{ matrix.compile }} appcompat: ${{ matrix.appcompat }}"
+    needs: [build]
```

Comment:
Why does this need the `build` step? It looks like it can run in parallel with the `build` step.